### PR TITLE
Reorg nomination pools

### LINKED ISSUE #3664: Reorg nomination pools
Removal of how-to guides, addition links to support pages, general reorg of the page if needed

### PR DESCRIPTION
**Pull request by filippoweb3**
Remove How-to guides, addition of links to support pages, general reorg of the page

### CODE REVIEW

File: docs/learn/learn-nomination-pools.md [hunk 1]
```diff
@@ -93,42 +88,20 @@ pool's internal logic can access the account.
...
 :::
 
-Navigate to Network > Staking > Pools page on Polkadot JS Apps UI and choose the pool to join. Click
-on the Join button and then sign and submit the transaction specifying the amount you would like to
-nominate.
-
-![Join Pool](../assets/staking/Nomination-Pools-3.png)
-
-![Join Pool](../assets/staking/Nomination-Pools-4.png)
+For guidelines about how to join a nomination pool visit [this](https://support.polkadot.network/support/solutions/articles/65000181401-how-to-join-nomination-pools) support article.
```
Comment (profradha) [inline docs/learn/learn-nomination-pools.md]:
```suggestion
Check the "How to join a pool" section in [this support article](https://support.polkadot.network/support/solutions/articles/65000181401-how-to-join-nomination-pools) for guidelines.
```

File: docs/learn/learn-nomination-pools.md [hunk 2]
```diff
@@ -93,42 +88,20 @@ pool's internal logic can access the account.
...
 The member can claim their portion of any rewards that have accumulated since the previous time they
 claimed (or in the case that they have never claimed, any rewards that have accumulated since the
-era after they joined). Rewards are split pro rata among the actively bonded members.
-
-On Polkadot JS Apps UI, navigate to Network > Staking > Accounts > Pooled, where you can view your
-pooled member account details. To claim rewards, click on the three vertical dots and click on
-Withdraw claimable.
-
-![Claim rewards](../assets/staking/nomination-pools-claim.png)
+era after they joined). Rewards are split pro rata among the actively bonded members. For guidelines about how to claim rewards in a nomination pool visit [this](https://support.polkadot.network/support/solutions/articles/65000181401-how-to-join-nomination-pools) support article.
```
Comment (profradha) [inline docs/learn/learn-nomination-pools.md]:
```suggestion
era after they joined). Rewards are split pro rata among the actively bonded members. Check the "How to claim rewards" section in [this support article](https://support.polkadot.network/support/solutions/articles/65000181401-how-to-join-nomination-pools) for guidelines.
```

File: docs/learn/learn-nomination-pools.md [hunk 3]
```diff
@@ -93,42 +88,20 @@ pool's internal logic can access the account.
...
-### Withdraw unbonded funds
-
-After `unbond` has been called and the unbonding duration has passed (28 eras {{ polkadot: which correspond to 28 days on Polkadot :polkadot }}{{ kusama: which correspond to 7 days on Kusama :kusama }}), a
+`unbond` will unbond part or all of the member's funds. After unbond has been called and the unbonding duration has passed (28 eras {{ polkadot: which correspond to 28 days on Polkadot :polkadot }}{{ kusama: which correspond to 7 days on Kusama :kusama }}), a
 member may withdraw their funds with `withdrawUnbonded`. Withdrawing effectively ends a member's
-relationship with their pool, allowing them to join a different pool if desired.
-
-On Polkadot JS Apps UI, navigate to Network > Staking > Accounts > Pooled and click on the three
-vertical dots and click on Withdraw unbonded.
+relationship with their pool, allowing them to join a different pool if desired. For guidelines about how to unbond and withdraw funds in a nomination pool visit [this](https://support.polkadot.network/support/solutions/articles/65000181401-how-to-join-nomination-pools) support article.
```
Comment (profradha) [inline docs/learn/learn-nomination-pools.md]:
```suggestion
relationship with their pool, allowing them to join a different pool if desired. Check the "Withdraw unbonded funds" section in [this support article](https://support.polkadot.network/support/solutions/articles/65000181401-how-to-join-nomination-pools) for guidelines.
```

**Review by DrW3RK — APPROVED** [review]
Looks good. Made a few suggestions.

**Pull request merged**